doc/iterations/2025.2/add_project_elements_selection_to_urls.adoc

```diff
@@ -0,0 +1,89 @@
+
+=== Scenario 1 : Obtaining the URL from a workbench selection
+
+- The end user is in a project.
+- If no representation is opened, the URL looks like `+https://example.org/projects/project-id/edit+`.
+- If a representation is opened, the URL looks like `+https://example.org/projects/project-id/edit/representation-id+`.
+
+The following behavior is the same in both cases.
+
+- The end user selects project elements.
+This can be done through any section of the workbench that supports selections: _Explorer_ view, _Representation Editor_, _Related Elements_ view, etc.
+
+- The selected elements can be:
+-- _Models_, e.g. the root elements in the _Explorer_ view.
+-- _Representations_, e.g. the _Explorer_ view displays them below their target element.
+-- _Semantic elements_, contained by a _model_ or a _semantic element_.
+-- _Graphical elements_, i.e. part of a representation.
+In that case, the intention is to capture in the selection their target semantic elements.
+
+- Upon selecting an element in the workbench, its corresponding project element identifier is retrieved, e.g. `+id_1+`.
+- The URL gets updated automatically in order to reflect the selection change, into either: 
+-- `+https://example.org/projects/project-id/edit?selection=id_1+`
+-- `+https://example.org/projects/project-id/edit/representation-id?selection=id_1+`
+- Upon selecting another element in the workbench, its corresponding project element identifier is retrieved and appended to the _selection_ parameter, which means the URL is updated to either:
+-- `+https://example.org/projects/project-id/edit?selection=id_1&id_2+`
+-- `+https://example.org/projects/project-id/edit/representation-id?selection=id_1&id_2+`
+- If selecting two graphical elements representing the same target semantic element, the following URLs are also possible:
+-- `+https://example.org/projects/project-id/edit?selection=id_1&id_1+`
+-- `+https://example.org/projects/project-id/edit/representation-id?selection=id_1&id_1+`
+
+=== Scenario 2 : Resolution of a URL containing the selection parameter
+
+- The URL path, whether it is a project or representation URL, is resolved as usual.
+  This means the specified project is opened, and if a representation is specified, it is opened in the representation editor.
+
+- The identifiers specified in the _selection_ parameter of the URL query are retrieved.
+  Based on these identifiers, we are able to retrieve the project elements that are meant to be selected.
+
+- The workbench selection is set to those elements.
+
+- The selection is revealed in the workbench.
+
+=== Breadboarding
+
+N/A
+
+=== Cutting backs
+
+N/A
+
+== Rabbit holes
+
+- When setting up the workbench selection programmatically, we have to make sure that as a result of the selection changing, the URL does not get updated automatically.
```

Comment:
That's completely opposite to the current or expected behavior. The URL is synchronized with the selection, if the selection changes by any mean, the URL should change and the other way around. Countless of web application are behaving like that, it's one of the core goal of the URL, it encodes a state of the application.